README.md

```diff
@@ -14,6 +14,22 @@ ember install ember-feature-flags
 
 ### Usage
 
+#### Computed-Property Macro
+
+This addon provides a computed-property macro that you can use to add properties that toggle
+whenever feature-flags do:
```

Comment:
Elsewhere in the README, "feature flags" do not have a dash. I don't really care which way so long as the entire README is consistent.